### ISSUE TITLE
Add the performance and accessibility testing section to the readme

### ISSUE THREAD

**Issue by elielijah321** [elielijah321/ASE-Sum-2-Square-Root-Checker#10]
As a user, I want to see performance and accessibility testing results in the readme file so that I can understand how well the application performs and whether it is accessible to everyone.

**Acceptance Criteria:**
- The performance and accessibility testing results should be easy to understand
- The performance and accessibility testing results should cover all the key features of the application
- The performance and accessibility testing results should be included in the readme file
